src/main/java/org/folio/innreach/util/JsonHelper.java

```diff
@@ -53,4 +58,17 @@ public <T> T fromJson(InputStream inputStream, Class<T> valueType) throws IOExce
     return obj;
   }
 
+
+  public static Long getCheckOutTimeDuration(List<ConfigurationDTO> configData) {
+    log.info("JsonHelper:getCheckOutTimeDuration started");
+    long checkOutTime = 1;
+    if(!configData.isEmpty()) {
+      var value = configData.get(0).getValue();
+      JsonObject valueObject = new Gson().fromJson(value, JsonObject.class);
+      checkOutTime = valueObject.get(CHECKOUT_TIMEOUT_DURATION).getAsLong();
+    }
+    log.info("JsonHelper:getCheckOutTimeDuration completed");
+    return (checkOutTime * 60000);
```

Comment:
It looks like we're working with units of milliseconds. That's fine but the code could be easier to follow if that were more clear. Perhaps renaming this method to `getCheckoutTimeDurationInMilliseconds` would help.

Reply:
Also you don't need the `(` and `)` around this statement.

Reply:
done